tools/vsextension/ProjectTemplates/oeenclave-linux/MyEnclave.Debug.conf

```diff
@@ -0,0 +1,8 @@
+# Enclave settings.  Any uncommented-out values will override what
+# is defined in the code.  The values below are all SGX specific.
+#Debug=1
```

Comment:
The recommended workflow is to use the conf file and not embed the enclave configuration properties in code.
Do we want to encourage that?

Reply:
I see no such recommendation in any docs.   Since OPTEE parameters cannot be put in the .conf file, and since it would be confusing for OPTEE and SGX parameters to be in different places, they both go in the code.   The conf file can override the code if a signer wants to override the developer's choices, but here these templates are for the developer.

Reply:
Issue #2714 is related to this point.